indico/htdocs/js/indico/jquery/categorynavigator.js

```diff
@@ -28,8 +28,22 @@
             openInDialog: false,
             // The title for the category navigator dialog
             dialogTitle: $T.gettext("Select a category"),
-            // Restrict action to categories with no subcategories
-            selectLeafOnly: false,
+            // Allow action to specific categories
+            actionOn: {
+                categoriesWithSubcategories: {
+                    disabled: false,
+                    message: $T.gettext("You cannot select a category that contains subcategories")
+                },
+                categoriesWithEvents: {
+                    disabled: false,
+                    message: $T.gettext("You cannot select a category that contains events")
+                },
+                self: {
+                    disabled: false,
+                    message: $T.gettext("You cannot select the same category"),
```

Comment:
I think the text `"You cannot perform the operation on this category"` is to "technical" for the end user. I think the word `"select"` is a good choice, because you can **select** something for deletion, you **select** a target category when you move something, you **select** a category to `"perform the operation"`